bql/semantic/convert.go

```diff
@@ -69,6 +69,11 @@ func (c ConsumedElement) Token() *lexer.Token {
 	return c.token
 }
 
+// Token returns the boxed token.
```

Comment:
String returns. ...

Reply:
I am not sure how I let it pass. Thank you for pointing out Jose.